src/pip/_internal/commands/freeze.py

```diff
@@ -86,6 +87,13 @@ def run(self, options, args):
 
         cmdoptions.check_list_path_option(options)
 
+        if options.find_links:
+            deprecated(
+                "--find-links option in pip freeze is deprecated.",
+                replacement=None,
+                gone_in="21.2",
+            )
```

Comment:
Let's add the issue number here, so that folks know where to make noise if they care? #9069 would be the one I'd add.

Reply:
Indeed! Done.